### DISCUSSION TITLE
Hover text on budget screen is misleading

### DISCUSSION THREAD

**Discussion by death2cupbots** [firefly-iii/firefly-iii#4317] [Firefly III]
**Bug description**
I am running Firefly III version 5.4.6, and my problem is:
When you make a budget and give it an amount that is not a whole number (e.g. 150.21), if you hover over that budget, it gives you hover text that reads "Please enter a valid value. The nearest valid values are x and y." x and y are the closest whole numbers (e.g. 150 and 151).
![Screenshot from 2021-01-18 22-16-37](https://user-images.githubusercontent.com/21091510/104983753-6446bf00-59db-11eb-80f4-b58f95179a55.png)

This seems misleading because firefly iii works just fine with the non-whole number value.

**Steps to reproduce**

1. Create a budget with a with a non-whole number for the budgeted amount. (e.g. 150.21)
2. Hover over the budget.
3. Notice the hover text.

**Extra info**

Debug information generated at 2021-01-18 22:24:38 America/New_York for Firefly III version **5.4.6**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.6 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.13 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 33fe0fbe-cfee-496b-81fa-a926233790d2 |
| Using docker? | false |
| Telemetry | false |
| Layout |  |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql |
| Current driver | mysql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2021-01-01 00:00:00 |
| Session end | 2021-01-31 23:59:59 |
| Session first | 2019-06-17 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36 |

**Bonus points**

- [x] I searched and nobody reported this bug before
- [ ] I have added a stack trace from my log files 
- [x] I have added a screenshot.
- [x] I was able to replicate it on the demo site https://demo.firefly-iii.org/

**Comment by JC5** [comment]
I'm pretty sure that's your browser, not me. I'll check it but I'm 99% sure I never wrote something like that.

**Comment by death2cupbots** [comment]
Ok thanks! It's happening to me on Brave, Firefox, and Vivaldi on Linux and Brave and Firefox on Windows.

**Comment by JC5** [comment]
It's your browser, I see it too in Firefox but the text isn't present in the code. There's also no "step=1" which could explain it. Sorry, I'm not sure if it can be disabled.

**Comment by death2cupbots** [comment]
Ok that's fine. Thanks for looking into it! At least it's not breaking anything.
